marshal/lib/config.py

```diff
@@ -22,21 +22,118 @@
 #  OTHER DEALINGS IN THE SOFTWARE.
 #
 
-## TODO: Allow overrides from ENV
+"""Simple configuration module that allows environment overrides"""
+
+import os
+
 
 KV = dict()
+_env_prefix = 'SBPS_MARSHAL_'
+
+# How often the agent should sleep between scans for projection changes (seconds)
+
+if os.environ.get(_env_prefix + 'SCAN_FREQUENCY') is not None:
```

Comment:
Just a query rather than a comment. I think the usage of os.environ.get() is intentional instead of using wrapper function os.getenv() right ? I think its due to os.environ.get() returns None if variable is not found whereas os.getenv() returns default value which we don't want. Is my understanding correct ?

Reply:
See https://docs.python.org/3/library/os.html#os.getenv, and note the following behavior

```
>>> os.environ.get('foo') is None
True
>>> os.getenv('foo') is None
True
```

Reply:
Thanks!